pkgs/tools/admin/acme.sh/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchFromGitHub }:
+stdenv.mkDerivation rec {
+  name = "acme.sh-${version}";
+  version = "2.7.8";
+
+  src = fetchFromGitHub {
+    owner = "Neilpang";
+    repo = "acme.sh";
+    rev = version;
+    sha256 = "0zm64z7av63xi7yjhljab2i8q1vx4q1mpcmcm58jm6k4babalxrf";
+  };
+
+  phases = [ "installPhase" ];
+
+  installPhase = ''
+    mkdir -p $out $out/bin $out/lib
+    cp -R $src/* $_
+    ln -s $out/lib/acme.sh $out/bin
```

Comment:
Should be wrapped with `wrapProgram` to have openssl and curl in path: https://github.com/Neilpang/acme.sh/blob/master/acme.sh#L31